manage/filters.js

```diff
@@ -76,6 +76,16 @@ onDOMready().then(onBackgroundReady).then(() => {
     }
   });
 
+  // Adjust width after selects are visible
+  $('#filters').addEventListener('click', event => {
```

Comment:
1. no need to debounce if it's not open so check it first
2. no need for `closest` if you use a `function` and `this`
3. the state may change due to external change in prefs so use `prefs.subscribe` instead of click listener